src/ARCValidationPackages/Config.fs

```diff
@@ -3,44 +3,68 @@ open System.IO
 open System.Text.Json
 
 type Config = {
-    PackageIndex: ValidationPackageIndex []
-    IndexLastUpdated: System.DateTimeOffset
+    PackageIndex: ValidationPackageIndex [] option
+    IndexLastUpdated: System.DateTimeOffset option
     PackageCacheFolder: string
     ConfigFilePath: string
+    IsPreview: bool
```

Comment:
@caroott please note that it must be possible to install both preview and prod packages side-by-side.

This looks to me like that use case would be problematic